user-guide/Standard_Apps/IDP/Tutorials/IDP_Tutorial_DiscoveryAndProvisioning.md

```diff
@@ -0,0 +1,160 @@
+---
+uid: IDP_Tutorial_DiscoveryAndProvisioning
+keywords: idp tutorial, idp kata
+---
+
+# Getting started with IDP and onboarding new equipment
+
+This tutorial will show you how to install IDP and onboard new equiment.
```

Comment:
Add that we will discovery & provision the server in DataMiner. Without specifying the end goal, users don't know why they are doing the steps.

Reply:
Added